app/src/docs/v1.api-spec.yaml

```diff
@@ -935,13 +929,17 @@ components:
       schema:
         type: boolean
         example: true
-    Query-Tag:
+    Query-TagSet:
       in: query
-      name: anyKey
-      description: Tags to add to the object, defined as Key/Value pairs
+      name: tagging[*]
+      description: >-
+        Tags to add to the object, defined as Key/Value TagSet. The query should
+        be formatted in deepObject style notation, where multiple TagSets would
+        be encoded something similar to `tagging[x]=a&tagging[y]=b`.
```

Comment:
maybe write this as:
'An array of key/value tags to add to the object. The query should be formatted in deepObject style notation `tagset[x]=a&tagset[y]=b`.'

I'm not quite sure why you used tagging[*]  for the name on line 934

Reply:
Fixed